### ISSUE TITLE
fanboy-annoyance.txt blocks everything

### ISSUE THREAD

**Issue by runerune** [ryanbr/fanboy-adblock#1384]
### List the website(s) you're having issues:

All of them, and in case some bot reviews this:
https://google.com
https://facebook.com
https://apple.com
https://microsoft.com

### What happens?

Everything gets blocked by filter "*"

### List Subscriptions you're using:

https://secure.fanboy.co.nz/fanboy-annoyance.txt?a=0

### Your settings

- OS/version: Android 10
- Browser/version: Firefox 68.10.1
- Adblock Extension/version: Ublock Origin 1.27.10

### Other details:

Caused by this:

```
dplay.se##div[class^="notification"]
!
! ---------- Thai ----------
!
pantip.com##.pt-snackbar__surface
|                                            <--- this pipe here
! ---------- Turkish ----------
!
find.com.tr###KVK_Cerez
```

FYI: this change is not in the repo.

**Comment by runerune** [comment]
If you are parsing your own lists. this can be removed with regex `/^\|$\n/`

**Comment by peterrakmanyi** [comment]
Came here to report the same thing. New update broke stuff.
@runerune Nice job on finding that pipe though, that is a funny bug.

Can't wait until the some critical infrastructure goes down because of a missing semicolon in a seemingly unrelated software package. :)

**Comment by peterrakmanyi** [comment]
Looks like it's already fixed.

**Comment by SuperHirmu** [comment]
Fixed from EasyList Cookie list, but not fanboy-annoyance. Version 202007171059 blocks everything.

**Comment by runerune** [comment]
It is fixed in the newest 202007171141 version, probably a local caching issue on your @SuperHirmu end and should resolve within minutes.

I wonder how did that pipe get in there, those headers should be generated, looks like someone messed with it manually?

**Comment by peterrakmanyi** [comment]
@SuperHirmu Would it make sence to parse outside lists instead of using them as-is?

Even though this looks like an innocent mistake, it shows how vulnarable everything is without checks in place before accepting an update. The more everything relies on each other, the more things can be sabotaged if things are not caught by a sanity check.
